src/main/java/uk/gov/pay/adminusers/service/GovUkPayAgreementService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.pay.adminusers.service;
+
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import uk.gov.pay.adminusers.logger.PayLoggerFactory;
+import uk.gov.pay.adminusers.model.GovUkPayAgreement;
+import uk.gov.pay.adminusers.persistence.dao.GovUkPayAgreementDao;
+import uk.gov.pay.adminusers.persistence.entity.GovUkPayAgreementEntity;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import static java.lang.String.format;
+
+public class GovUkPayAgreementService {
+
+    private static Logger logger = PayLoggerFactory.getLogger(GovUkPayAgreementService.class);
```

Comment:
Probably want this to be `final` as well (and the `LOGGER` because it’s a constant)

Reply:
Fair dinkum